pages/reference.mdx

```diff
@@ -634,3 +634,7 @@ These requests do not require the `Authorization` header to be present.
 ##### OAuth2 Request
 
 The endpoint may be used with a bearer token in the `Authorization` header. The specific scope required (if any) will be provided if you hover over the badge.
+
+##### Deprecated Endpoint
+
+This endpoint remains active and functionally the same, but it should be avoided if at all possible as it may be removed in a future API version.
```

Comment:
```suggestion
This endpoint remains active and functionally the same, but its further existence is not guaranteed. It may be removed in a future API version, removed entirely, or not removed at all.
```